docs/releases/changelog-dev.md

```diff
@@ -2,9 +2,20 @@
 
 ### New features since last release
 
-### Improvements 
+- Added the Qblox-specific `set_markers()` method in `QProgram`. This method takes a 4-bit binary mask as input, where `0` means that the associated marker will be open (no signal) and `1` means that the associated marker will be closed (signal). The mapping between bit indexes and markers depends on the Qblox module that the compiled `QProgram` will run on.
 
-### Breaking changes  
+  Example:
+
+  ```Python
+  qp = QProgram()
+  qp.qblox.set_markers(bus='drive_q0', mask='0111')
```

Comment:
Is it relevant where in the QProgram declaration this instruction is located? Is it beneficial to do it at the very beginning on any order is fine, depending on the bus? If so, could we indicate it into this example? that will help when creating documentation later on and for the experimentalists for the time being.

Reply:
According to Adrian the idea is to set_markers at the beggining of the program and at the end, and never inside any loop.

Reply:
Cool, could you please @fedonman specify that in this example?

Reply:
It's not wrong to set markers inside a loop. Qblox has practical examples that do exactly that. Eg. in the Rabi: https://qblox-qblox-instruments.readthedocs-hosted.com/en/main/applications/q1asm/rabi_experiment.html

Reply:
Sorry for being annoying, could we synthesise all this in the example? A line with your last comment would suffice for example.

Reply:
What exactly do you want to add?

Reply:
I wanted to try have some minimum guidelines that could substitute proper documentation or a demo for a bit, nothing too serious, but I would not delay this much more just for it, so feel free to resolve the conversation if you fancy.

Reply:
As a minidemo I would like to try to implement the markers with adrian and understand what happens and how it works

Reply:
Be my guess if you fancy taking some time off your tasks and work on a demo, it would be nice to have.

Reply:
I guess this will be a different PR right? It can be combined with qprogram documentation task.